samza-kafka/src/main/java/org/apache/samza/system/kafka/KafkaSystemConsumer.java

```diff
@@ -265,28 +259,39 @@ public void stop() {
    */
   @Override
   public void register(SystemStreamPartition systemStreamPartition, String offset) {
-    register(systemStreamPartition, new StartpointSpecific(offset));
-  }
-
-  @Override
-  public void register(SystemStreamPartition systemStreamPartition, Startpoint startpoint) {
     if (started.get()) {
-      String exceptionMessage = String.format("KafkaSystemConsumer: %s had started. Registration of ssp: %s, startpoint: %s failed.", this, systemStreamPartition, startpoint);
+      String exceptionMessage = String.format("KafkaSystemConsumer: %s had started. Registration of ssp: %s, offset: %s failed.", this, systemStreamPartition, offset);
       throw new SamzaException(exceptionMessage);
     }
 
     if (!Objects.equals(systemStreamPartition.getSystem(), systemName)) {
       LOG.warn("{}: ignoring SSP {}, because this consumer's system doesn't match.", this, systemStreamPartition);
       return;
     }
+    LOG.info("{}: Registering ssp = {} with offset {}", this, systemStreamPartition, offset);
```

Comment:
nit: `LOG.info("{}: Registering ssp: {} with offset: {}", this, systemStreamPartition, offset);`
for logging consistency